FirebaseAuth/Sources/Swift/Backend/IdentityToolkitRequest.swift

```diff
@@ -30,47 +30,63 @@ private let kIdentityPlatformStagingAPIHost =
 
 /// Represents a request to an identity toolkit endpoint.
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-open class IdentityToolkitRequest {
```

Comment:
I'm curious why these keywords are no longer needed?

Reply:
Extra `public` visibility was necessary for porting when the library was a mix of Objective C and Swift.  Now that it is all Swift, the access can be corrected. 

A full review of all the access specifiers across all the sources is on the TODO list.